learning-development/accessibility.qmd

```diff
@@ -9,10 +9,15 @@ title: "Accessibility"
 
 ## Accessibility in DfE
 
+Everyone has a part to play in ensuring that the content and digital services the DfE provides to it's customers and colleagues are accessible. Everyone should have an understanding of their responsibilities to comply with the [Public Sector Bodies Accessibility Regulations 2018](https://www.legislation.gov.uk/uksi/2018/952/contents/made) and the [Web Content Accessibility Guidelines 2.2](https://www.w3.org/TR/WCAG22/). Moreover, everyone should also understand the important and critical difference this makes to the everyday lives of millions of people.
```

Comment:
Should be "its customers" not "it's customers" - sorry, very pedantic

Reply:
Do not apologise for this - it's a mistake that wants fixing!

Reply:
Updated in PR response 2 commit